util/src/main/i18n/templates/match.properties

```diff
@@ -152,6 +152,13 @@ match.timeLimit.result = {1} after {0}
 # {1} = some criteria to win the match
 match.timeLimit.commandOutput = The time limit is {0} with the result {1}
 
+match.timeLimit.notEnabled = The time limit is not available on this map
+
+# {0} = new score limit of the match
+match.scoreLimit.commandOutput = The score limit is now {0}
+
+match.scoreLimit.notEnabled = The score limit is not available on this map
```

Comment:
"**The** score limit" sounds weird to me, maybe something like

`This map does not use score so a score limit can not be set` or just `This map does not use score`?